src/citrine/informatics/design_candidate.py

```diff
@@ -45,7 +45,9 @@ class MeanAndStd(Serializable["MeanAndStd"], DesignVariable):
     """
 
     mean = properties.Float('m')
+    """:float: mean of the continuous distribution"""
```

Comment:
in general, I am used to the comments going above the line they are commenting , is there a reason all of these are below them?